src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationUtils.java

```diff
@@ -149,4 +145,23 @@ public static final boolean hasSubscription(NotificationWrapper wrapper, Subscri
             || APPOINTEE.equals(subscriptionType)
             || (REPRESENTATIVE.equals(subscriptionType) && null != wrapper.getNewSscsCaseData().getAppeal().getRep());
     }
+
+    public static final Hearing getLatestHearing(SscsCaseData sscsCaseData) {
+        List<Hearing> hearings = new ArrayList<Hearing>(sscsCaseData.getHearings());
+
+        Comparator<Hearing> compareByIdAndDate = new Comparator<Hearing>() {
+            @Override
+            public int compare(Hearing o1, Hearing o2) {
+                Integer idCompare = o1.getValue().getHearingId().compareTo(o2.getValue().getHearingId());
+                if (idCompare != 0) {
+                    return -1 * idCompare;
+                }
+                return -1 * o1.getValue().getHearingDateTime().compareTo(o2.getValue().getHearingDateTime());
+            }
+        };
+
+        Collections.sort(hearings, compareByIdAndDate);
+
+        return hearings.get(0);
```

Comment:
Would it throw an exception if there is an empty list?

Reply:
You're right - in this particular case, the check is performed before the call to the method, but I've updated the PR to return null from the method in that situation.